render/present.go

```diff
@@ -0,0 +1,100 @@
+package render
```

Comment:
I think rename this file "presenter.go" since it better reflects the naming in the code